test/unit/application_validator_test.rb

```diff
@@ -39,24 +39,6 @@ class ApplicationValidatorTest < Minitest::Test
       refute result.ok?, result.error_value
     end
 
-    test "returns error for mismatched inflections.yml file" do
```

Comment:
This test case seems to be validating the bug.

Previously `custom_inflections.yml` was not specified in the skeleton configuration file that is used for our integration tests. In this test, the file was added to the configuration and expected to fail. This was because the validator was using the `ActiveSupport::Inflector` and not checking for any custom inflections.

Reply:
Again, an excellent piece of documentation that deserves to live in the commit message.